src/Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs

```diff
@@ -119,18 +119,9 @@ protected override void ProcessRecord()
                     {
                         AlternateDataStreamUtilities.DeleteFileStream(path, "Zone.Identifier");
                     }
-                    catch (Win32Exception accessException)
+                    catch (Exception accessException)
```

Comment:
Should the variable be renamed to `exception`?

Reply:
I agree that the name should be replaced (if we'll keep the try-catch). I did search in all files - usually we use "catch (Exception e)" pattern.